### ISSUE TITLE
Duplicate results on experimental GET /documenten/api/v1/enkelvoudiginformatieobjecten?objectinformatieobjecten__object=....

### ISSUE THREAD

**Issue by Klaas-Ritense** [open-zaak/open-zaak#1653]
### Product versie / Product version

1.13.0

### Omschrijf het probleem / Describe the bug

The new experimental endpoint can return multiple versions of the same enkelvoudiginformatieobject when sorting on `creatiedatum`.

### Stappen om te reproduceren / Steps to reproduce

1. Create a enkelvoudiginformatieobject with endpoint `POST /documenten/api/v1/enkelvoudiginformatieobjecten`
2. Update the newly created enkelvoudiginformatieobject with endpoint `PUT /documenten/api/v1/enkelvoudiginformatieobjecten/{uuid}`
3. Get a list of all enkelvoudiginformatieobject with endpoint `GET /documenten/api/v1/enkelvoudiginformatieobjecten?objectinformatieobjecten__object={url}&ordering=-creatiedatum`
4. Check that you ordered on `creatiedatum`
5. Notice that both versions of the enkelvoudiginformatieobject are returned

### Verwacht gedrag / Expected behavior

We expected the endpoint to return only the latest version of the enkelvoudiginformatieobject.

<img width="811" alt="Screenshot 2024-05-13 at 16 25 53" src="https://github.com/open-zaak/open-zaak/assets/94360980/2f975f69-f0b2-4f5f-be73-4b617da998ae">

**Comment by joeribekker** [comment]
Refinement: Steps to reproduce seem to miss linking to an object or zaak but otherwise clear.

Estimate: 3 days to fix

**Comment by joeribekker** [comment]
@JanBrek

**Comment by joeribekker** [comment]
@JanBrek ping

**Comment by alextreme** [comment]
@janmasGDH

**Comment by JanBrek** [comment]
@alextreme Do we have an origin email / ticket? I would like to find out the priority of the initial bug reporter to get this fixed.

**Comment by alextreme** [comment]
@JanBrek no, this didn't come in via me. I'd recommend to double-check with Thomas / Ritense
